### PR TITLE
feat(UIS-49):Update employee

### PR DESCRIPTION
**Pull request by StaceyJin**
JIRA: https://champlainsaintlambert.atlassian.net/browse/UIS-49

Update employee functionality implemented in frontend, story is COMPLETED:

- functional component added to user interface;
- field validation applied to form;
- end to end testing via Cypress implemented.

### CODE REVIEW

File: src/components/employeeDetails.vue [hunk 1]
```diff
@@ -0,0 +1,105 @@
+<template>
+  <br>
+  <h1 class="text-center">UNIVERSAL MARKETING CO., LTD</h1>
```
Comment (Kam1st) [inline src/components/employeeDetails.vue:3]: This was updated in previous merges to have the title in the App. Vue. We can take those off.

File: package.json [hunk 2]
```diff
@@ -9,6 +9,9 @@
   },
   "dependencies": {
     "@auth0/auth0-vue": "^1.0.3",
+    "@vuelidate/core": "^2.0.0",
```
Comment (Kam1st) [inline package.json:12]: What are those new dependencies for?

**Review by Kam1st — APPROVED** [review]
Otherwise looks fine.

File: package.json (hunk 2 above)
Comment (StaceyJin) [reply package.json:12]: A library for validation, but it didn't end up being used, will be removed.

File: src/components/employeeDetails.vue (hunk 1 above)
Comment (StaceyJin) [reply src/components/employeeDetails.vue:3]: Will update and remove title.

**Pull request merged**